resources/usr/bin/openhab-cli

```diff
@@ -55,10 +56,10 @@ case "$option" in
     fi
 
     # Find the appropriate daemon tool
-    if [ -x "/bin/systemctl" ]; then
+    if [ -x "/bin/systemctl" ] && [ -f "/usr/lib/systemd/system/openhab2.service" ]; then
       echo "A systemd service configuration exists..."
       echo "Use 'sudo /bin/systemctl $option openhab2.service' to $optionVerb an openHAB service"
-    else
+    elif [ -x "/etc/init.d/openhab2" ]; then
```

Comment:
👍 on both.
Do you think it's safe to go without an else clause? I'm a big fan of the `else raise UtterlyHorribleException` concept

Reply:
The two statements are only for informational puposes to the user, so I should think so. The intended purpose of `start` is to start a signal instance and it should do that regardless of the presence of those files right?

Reply:
Ah I see. In this case I'd agree.